kinder/pkg/kubeadm/config.go

```diff
@@ -66,13 +60,15 @@ func Config(kubeadmVersion *K8sVersion.Version, data ConfigData) (config string,
 	return buff.String(), nil
 }
 
-// getKubeadmConfigVersion returns the kubeadm config version corresponding to a Kubernetes kubeadmVersion
-func getKubeadmConfigVersion(kubeadmVersion *K8sVersion.Version) (string, error) {
-	// returns the corresponding config version
-	// nb v1alpha1 (that is Kubernetes v1.10.0) is out of support,
-	// v1alpha2 (that is Kuberntes v1.11.0) is out of support.
-	// v1alpha3 (that is Kuberntes v1.13.0) is out of support.
+// GetKubeadmConfigVersion returns the kubeadm config version.
+// It returns the kubeadmConfigVersion specified by user first when that is available.
+// Otherwise, returns the kubeadm config version corresponding to a Kubernetes kubeadmVersion.
+func GetKubeadmConfigVersion(kubeadmVersion *K8sVersion.Version, kubeadmConfigVersion string) (string, error) {
```

Comment:
why not remove `kubeadmVersion *K8sVersion.Version` from this function's arguments at this point + (also update the comment above the function)? it is not used.

Reply:
As this function has no complicated logic, I prefer to remove it completely.
We can just use a constant as the default kubeadm config version.